modules/engine/src/model/model.ts

```diff
@@ -489,9 +497,22 @@ export class Model {
    */
   setInstanceCount(instanceCount: number): void {
     this.instanceCount = instanceCount;
+    if (instanceCount > 0) {
+      this.isInstanced = true;
+    }
```

Comment:
I would expect this to work the other way, i.e.

```
if (this.isInstanced) {
  this.instanceCount = instanceCount;
}
```

Reply:
This was added as I didn't wan't to break existing luma.gl examples.